Framework/Core/test/benchmark_ASoAHelpers.cxx

```diff
@@ -629,7 +629,7 @@ static void BM_ASoAHelpersCombGenSimpleFivesSameCategories(benchmark::State& sta
   }
   auto table = builder.finalize();
 
-  using Test = o2::soa::Table<test::X>;
+  using Test = o2::soa::Table<o2::header::DataOrigin{"AOD"}, test::X>;
```

Comment:
I really do not like having the Table to specify the DataOrigin. In the long term (i.e. not this PR) we could maybe introduce a template alias so that this goes back to what it was. I appreciate this is not exposed to the users, but still..

Reply:
I agree, for the generic non-data-model table we need some default origin provided by an alias. I will make the change for the next pass.